server/service/src/sync/api_v6/mod.rs

```diff
@@ -115,6 +124,7 @@ pub struct SyncBatchV6 {
     // Including records in this batch
     pub(crate) total_records: u64,
```

Comment:
Haven't done anything about the total_records thing yet... not sure if it's better to rename or remove?